docroot/themes/humsci/humsci_basic/templates/menus/macros/nav-menu.twig

```diff
@@ -27,9 +27,26 @@
     {% endif %}
 
     <li{{ list_attributes }}>
-      <a{{ link_attributes }}>{{ item.title }}</a>
-      {% if item.below %}
-        {{ menus.nav_menu(item.below, menu_level + 1, class_prefix) }}
+      {% if item.url.toString() is empty %}
```

Comment:
can you explain this if check a bit more? I am not exactly sure how or why we decide between button or link

Reply:
or adding a comment would be awesome